proof/refine/AARCH64/CNodeInv_R.thy

```diff
@@ -4165,6 +4176,24 @@ lemma SGI_scap_neq_cap:
   using src_derived
   by (auto simp: weak_derived'_def isCap_simps)
 
+lemma isSMC_src:
+  "isArchSMCCap scap = isArchSMCCap src_cap"
+  using src_derived
+  by (fastforce simp: isCap_simps weak_derived'_def)
+
+lemma isSMC_dest:
+  "isArchSMCCap dcap = isArchSMCCap dest_cap"
+  using dest_derived by (fastforce simp: isCap_simps weak_derived'_def)
```

Comment:
one of these is not like the others (should we be wrapping the `by` on the same line, or saving space? either way, these four should be the same)